sdkv2.0.0/sdkv2.0.0/MojioSDK/MojioTokenManager.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MojioTokenManager.swift
+//  Motion
+//
+//  Created by Ashish Agarwal on 2016-03-10.
+//  Copyright © 2016 Mojio. All rights reserved.
+//
+
+import UIKit
+import KeychainSwift
+
+class MojioTokenManager: NSObject {
+    
+    private let MOJIO_AUTH_TOKEN_STRING = "MojioAuthToken"
+    private let MOJIO_REFRESH_TOKEN_STRING = "MojioAuthRefreshToken"
+    private let MOJIO_TOKEN_EXPIRES_STRING = "MojioAuthTokenExpiresIn"
+    
+    func getAuthToken () -> (String?, String?, NSString?) {
+        let keychain = KeychainSwift()
+        let authToken : String? = keychain.get(self.MOJIO_AUTH_TOKEN_STRING)
+        let refreshToken : String? = keychain.get(self.MOJIO_REFRESH_TOKEN_STRING)
+        let expiryDate : NSString? = keychain.get(self.MOJIO_TOKEN_EXPIRES_STRING)
+        
+        return (authToken, refreshToken, expiryDate)
+    }
+    
+    func saveAuthenticationToken (token : String, refreshToken : String, expiresIn : Double) -> Void {
```

Comment:
In Gauge, I was making an API call to /Users/Me to check and see if my authentication token was still valid. This approach is better. Have added it in.